packages/components/combobox/src/create-custom-option.ts

```diff
@@ -25,7 +25,7 @@ export class CreateCustomOption extends Option {
     return html`
       <div part="container">
         <sl-icon name="plus"></sl-icon>
-        <div part="wrapper">${msg(str`Create "${this.value}"`)}</div>
+        <div part="wrapper">${msg(str`Create "${this.value}"`, { id: 'sl.createCustomOption.createOptionLabel' })}</div>
```

Comment:
I'm not sure i would use `createCustomOption` as the "component" here. What about `sl.combobox.createCustomOption`?

Reply:
It would make it easier for the translator to know where to find the code.